packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -122,7 +123,8 @@ export interface VideoGalleryProps {
   onDisposeRemoteStreamView?: (userId: string) => Promise<void>;
   /** Callback to render a particpant avatar */
   onRenderAvatar?: OnRenderAvatarCallback;
-
+  /* @conditional-compile-remove-from(stable) */
```

Comment:
Please add a tag to this directive to make it easier to find: `/* @conditional-compile-remove-from(stable) meeting-composite <more-context?> */`